### PR TITLE
Fix inconsistencies for coder junit tester

### PR DESCRIPTION
**Pull request by pbochis**

### CODE REVIEW

File: controllers/mock.go [hunk 1]
```diff
@@ -370,8 +370,8 @@ func MockFrequentisChallenge(ctx context.Context, coduno *datastore.Key, w http.
 			Security:     0,
 			CodingSpeed:  0.4,
 		},
-		Templates: templateHelper(map[string][]string{"java": []string{"avl/Tests.java"}}),
-		Languages: []string{"java"},
+		Templates: templateHelper(map[string][]string{"javaut": []string{"avl/Tests.java"}}),
```
Comment (flowlo) [inline controllers/mock.go]: THE LANGUAGE OF THE CODE IS JAVA!!!!!

File: controllers/mock.go [hunk 2]
```diff
@@ -370,8 +370,8 @@ func MockFrequentisChallenge(ctx context.Context, coduno *datastore.Key, w http.
...
+		Languages: []string{"javaut"},
```
Comment (flowlo) [inline controllers/mock.go]: THE LANGUAGE OF THE CODE IS JAVA!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

Comment (pbochis) [inline controllers/mock.go]: Go skype chat

**Comment by flowlo** [comment]
Why did you merge this? This was the most definite way of not LGTMing possible.

**Pull request merged**